src/main/java/sas1/project/demo/model/Ship.java

```diff
@@ -153,6 +185,7 @@ public int getSpeed() {
     }
 
     public void setSpeed(int speed) {
+
```

Comment:
else if (speed>this.maxSpeed){
            throw new IllegalArgumentException();
        }
Der kan være tilfælde hvor this speed er større end max speed pga begrænsning af fart reduktion ?

Reply:
if(currentSpeed > ship.getMaxSpeed()){
            currentSpeed = ship.getMaxSpeed();
        }
        return currentSpeed;
    } // Hvis Speed overskrider maxSpeed vil if statement ovenover sætter speed til maxSpeed